src/ucs/sys/topo/base/topo.c

```diff
@@ -283,6 +283,7 @@ ucs_topo_read_device_numa_node(const ucs_sys_bus_id_t *bus_id)
 ucs_status_t ucs_topo_find_device_by_bus_id(const ucs_sys_bus_id_t *bus_id,
                                             ucs_sys_device_t *sys_dev)
 {
+    ucs_status_t status = UCS_OK;
```

Comment:
IMO we should set status explicitly to UCS_OK when reaching `ucs_debug("added sys_dev...`